CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+
+## v0.33.9
+- Upgrade Ubuntu version to 20.04 to be current. 
+=======
 ## next
```

Comment:
```suggestion
## next

- Updated the how to get started with AWS section in documentation
```

Minor suggestion to fix the duplicate `v0.33.9` header and adding your change to the CHANGELOG if you think it should be added